_data/simple-icons.json

```diff
@@ -6431,6 +6421,12 @@
             "source": "https://openwrt.org/docs/guide-graphic-designer/openwrt-logo",
             "guidelines": "https://openwrt.org/docs/guide-graphic-designer/openwrt-logo"
         },
+        {
+            "title": "OpenSea",
+            "hex": "2081E2",
+            "source": "https://docs.opensea.io/docs/logos",
+            "guidelines": "https://docs.opensea.io/docs/logos"
```

Comment:
While I'm not necessarily against including it here, I feel like it's a bit of stretch to call this page "Brand guidelines" - @simple-icons/maintainers do we have any consensus on when to use the `"guidelines"` field in cases like this?

Reply:
I was just following what the other icons have done, most seem logos seem to be on the same page as the company explaining their brand.

Reply:
As I already pointed out, we (intentionally) didn't explicitly define what is and isn't allowed to be in the `"guidelines"` field. So, I'm not saying this is necessarily wrong.

For me an important factor is whether or not it includes guidelines on how one _should_ use the logo. Whereas this page seems to just give some logos and colors to use.